packages/sidenav/src/Sidenav.ts

```diff
@@ -81,6 +82,24 @@ export class SideNav extends Focusable {
     @property({ reflect: true })
     public variant?: 'multilevel' = undefined;
 
+    /**
+     * An accessible label that describes the component,
+     * so that the side navigation can be distinguished
+     * from other navigation by screen reader users.
+     * It will be applied to aria-label, but not visually rendered.
+     */
+    @property({ reflect: true })
+    public label?: string | undefined = undefined;
+
+    /**
+     * Identifies the element (or elements) that labels the component,
+     * so that the side navigation can be distinguished
+     * from other navigation by screen reader users.
+     * It will be applied to aria-labelledby, but not visually rendered.
+     */
+    @property({ reflect: true })
+    public labelledby?: string | undefined = undefined;
```

Comment:
If we absolutely what this type of accessor, we'd need to resolve the text content of any relating element and then apply that string to label, do the connections across the shadow boundary being blocked. Do you think that's necessary, or could we leave this property out of the API?

Reply:
We can omit `labelledby`. I actually had trouble associating the side nav on the docs with the Spectrum Web Components logo, because everything was contained within a web component.